WooCommerce/Classes/ViewRelated/Products/Edit Product/Edit Price/Bulk Edit Price/BulkUpdatePriceSettingsViewModel.swift

```diff
@@ -106,27 +117,31 @@ final class BulkUpdatePriceSettingsViewModel {
     /// Update the button state to enable/disable based on price value
     ///
     private func updateButtonStateBasedOnCurrentPrice() {
-        guard let price = currentPrice, price.isNotEmpty else {
-            saveButtonState = .disabled
+        // While the keyboard is in loading state do not change the state
```

Comment:
Did you mean button?
```suggestion
        // While the action button is a loading state do not change the state
```